spec/features/multi_play_spec.rb

```diff
@@ -0,0 +1,56 @@
+feature 'multiplayer' do
+  scenario 'visit front multiplayer mode and see the elements' do
+    visit '/'
+    click_button("3.  Player vs Player ")
+    expect(page).to have_content("ROCK PAPER SCISSORS LIZARD SPOCK!")
+  end
+  scenario 'visit front multiplayer mode and see the elements' do
+    visit '/'
+    click_button("3.  Player vs Player ")
+    expect(page).to have_content("Name yourself players!")
+  end
+  scenario 'visit front multiplayer mode and get back to menu' do
+    visit '/'
+    click_button("3.  Player vs Player ")
+    click_button("Back to Main Menu")
+    expect(page).to have_current_path("/")
+  end
+  scenario 'visit front nultiplayer mode and enter players names' do
```

Comment:
You can put this block of set up code in a spec helper and use it in the win, lose and draw scenarios to keep your code DRY